index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Not-Twitter</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="main-frame">
+      <header class="header">
+        <div id="title"><h1>Home</h1></div>
+
+        <img
+          id="toggle-icon"
+          src="./assets/toggle-feed-header.svg"
+          alt="Toggle feed"
+        />
+      </header>
+      <div class="left-side">
+        <img id="logo" src="./assets/logo-twitter.svg" alt="Logo" />
+        <div class="navigation">
+          <div class="nav-icons">
+            <img src="./assets/home-nav.svg" alt="Home icon" />
+            <img src="./assets/trending-nav.svg" alt="Exp" />
+            <img src="./assets/notifications-nav.svg" alt="Notification" />
+            <img src="./assets/messages-nav.svg" alt="" />
```

Comment:
nice use of svg in separate folder shows ur understanding of the lessons

Reply:
agreed. It makes the html code a lot less convoluted than sticking the entire svg path in (like I did lol)